gradle.properties

```diff
@@ -19,7 +19,7 @@
 #
 
 rootProject.name=fdb-record-layer
-version=3.2
+version=3.3
```

Comment:
I believe there are some more changes that are needed to properly update the minor version. For example, see how it was done in this commit: https://github.com/alecgrieser/fdb-record-layer/commit/1d17f61572db7465250733de34910c344da77802

In particular, the Release Notes should be updated to add a new header for the new minor version, and the `docs/Versioning.md` gets updated with the most recent version just to avoid that getting stale. It's also a good idea to add a paragraph to the new section so that it's easy for users to see what has changed.